v2/dtos/deviceprofile.go

```diff
@@ -62,13 +62,28 @@ func FromDeviceProfileModelToDTO(deviceProfile models.DeviceProfile) DeviceProfi
 // This object and its properties correspond to the UpdateDeviceProfile object in the APIv2 specification:
 // https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/UpdateDeviceProfile
 type UpdateDeviceProfile struct {
```

Comment:
These changes make `UpdateDeviceProfile` identical to `DeviceProfile`. Better to just remove `UpdateDeviceProfile`  and `UpdateDeviceProfileRequest` and rename `AddDeviceProfileRequest` to `AddUpdateDeviceProfileRequest` or just `DeviceProfileRequest`.

Reply:
No problem, removed the UpdateDeviceProfile and renamed the AddDeviceProfileRequest to DeviceProfileRequest.